src/actions/fetch.js

```diff
@@ -0,0 +1,120 @@
+import { createAction } from 'redux-actions';
+
+import { sleep } from '../util/debug.js';
+import { isArray } from '../util/types.js';
+import { isObject } from '../util/types.js';
+
+// replacement for redux-thunk-actions
```

Comment:
Can you also add the article's URL in the comments here? I found it very helpful to understand the code here.